src/build-config.ts

```diff
@@ -254,11 +270,10 @@ async function buildInputConfig(
       : [
           ...commonPlugins,
           preserveDirectives(),
-
           aliasPlugin,
           inlineCss({ exclude: /node_modules/ }),
           rawContent({ exclude: /node_modules/ }),
-
+          patchBinary(executablePaths),
```

Comment:
The rollup job is applied for each entry, so we can just pass the current entry file when it's a binary entry to the plugin. we don't need to collect all the entry paths, instead we only need to check if current entry is binary